pyQuARC/schemas/rule_mapping.json

```diff
@@ -3631,7 +3730,7 @@
     "get_data_url_check_umm": {
         "rule_name": "GET DATA URL check for UMM JSON",
         "fields_to_apply": {
-            "umm-json": [
+            "umm-g": [
```

Comment:
why'd you replace this? "umm-json" holds the ummc checks. by doing this, you are removing the ummc check and adding a ummg check. you probably want both checks, right? @jenny-m-wood

Reply:
@slesaad yep, that's a mistake on my part. Thanks for catching it!